### PR TITLE
Define postmortems introduction

### PR DESCRIPTION
**Pull request by DlHolland**
## Description

Added the postmortems introduction
### Type of Change

- [X] Feature
- [ ] Bug fix
- [ ] Documentation

## Contributors

- @DlHolland 

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

File: conduct.md [hunk 1]
```diff
@@ -100,13 +100,13 @@ TODO: Write an introductory paragraph.
 
 ### Postmortems
 
-TODO: Write an introductory paragraph.
+Postmortems are evaluation of performances after implementation. The goal is to identify what worked and what didn't and describe what needed to be done differently. When a postmortem is done correctly all team members should have a better understanding of how to avoid the same mistakes in future projects.
```
Comment (PaigeCD) [inline conduct.md]: In the first sentence "Postmortems are evaluation of performances", should this have been "are an evaluation" or "are evaluations" instead?

Comment (PaigeCD) [inline conduct.md]: Perhaps "didn't" should be "did not".

Comment (PaigeCD) [inline conduct.md]: In the last sentence, maybe add a comma between "done correctly" and "all team members".

**Review by PaigeCD — CHANGES_REQUESTED** [review]
The right sentiment is definitely there, but I have a few requested changes to polish it up!

**Comment by DlHolland** [comment]
> The right sentiment is definitely there, but I have a few requested changes to polish it up!

Thanks for pointing those out!  I went ahead and made the changes.  If you see anything else that needs work let me know!

**Review by mariakimheinert — APPROVED** [review]
Looks good, @DlHolland! Thanks! :shipit:

**Review by PaigeCD — APPROVED** [review]
Looks great! Thanks! 👍

**Pull request merged**